### ISSUE TITLE
No detection when using voc-weight

### ISSUE THREAD

**Issue by HrsPythonix** [marvis/pytorch-yolo2#136]
I download the cfg and weight from here:
https://pjreddie.com/darknet/yolov2/

When I use the coco model, everything goes well.
However, when I use the voc model, python detect.py cfg/yolo-voc.cfg yolov2-voc.weights data/dog.jpg, it detects nothing.

my pytorch version is 0.3.1, and it gives some warning:

UserWarning: src is not broadcastable to dst, but they have the same number of elements.  Falling back to deprecated pointwise behavior.
  conv_model.weight.data.copy_(torch.from_numpy(buf[start:start+num_w])); start = start + num_w

UserWarning: src is not broadcastable to dst, but they have the same number of elements.  Falling back to deprecated pointwise behavior.
  conv_model.weight.data.copy_(torch.from_numpy(buf[start:start+num_w])); start = start + num_w

UserWarning: Implicit dimension choice for softmax has been deprecated. Change the call to include dim=X as an argument.
  cls_confs = torch.nn.Softmax()(Variable(output[5:5+num_classes].transpose(0,1))).data

**Comment by HrsPythonix** [comment]
Fixed

**Comment by sxs11** [comment]
Did the warning matter? I also have these warnings, but it looks like having no influence on my VOC model. And I got the MAP 73.04

**Comment by JJZHK** [comment]
How to fix it? I met this issue......

**Comment by DeepPower** [comment]
@HrsPythonix Hi! I met the same error. What is your solution?
